<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>driveparams</class>
 <widget class="QWidget" name="driveparams">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Drive Parameter Setup</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>7</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>controller eeprom parameter setup</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveparam">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>287</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>&amp;Save to Controller</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>44</y>
     <width>486</width>
     <height>237</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_X">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_Y">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_Z">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_accel">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>acceleration</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_maxspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>maximum speed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_drv_para_X_maxspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_drv_para_Y_maxspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="spinBox_drv_para_Z_maxspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_softendpos">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>softend positiv</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="spinBox_drv_para_X_softend_pos">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="spinBox_drv_para_Y_softend_pos">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="spinBox_drv_para_Z_softend_pos">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_softendneg">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>softend negativ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="spinBox_drv_para_X_softend_neg">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="spinBox_drv_para_Y_softend_neg">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="spinBox_drv_para_Z_softend_neg">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_refp">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>reference pos</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spinBox_drv_para_X_refpos">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="spinBox_drv_para_Y_refpos">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="spinBox_drv_para_Z_refpos">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_refspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>reference speed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="spinBox_drv_para_X_refspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="spinBox_drv_para_Y_refspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QSpinBox" name="spinBox_drv_para_Z_refspeed">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_poscor">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>pos correction</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox_drv_para_X_poscor">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="spinBox_drv_para_Y_poscor">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QSpinBox" name="spinBox_drv_para_Z_poscor">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>-999999999</number>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_drv_para_X_accel">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_drv_para_Y_accel">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_drv_para_Z_accel">
      <property name="font">
       <font>
        <family>DejaVu Sans Mono</family>
        <pointsize>10</pointsize>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="maximum">
       <double>100000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_loadparam">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>287</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Load from Controller</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>287</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_drv_para_X_maxspeed</tabstop>
  <tabstop>spinBox_drv_para_X_softend_pos</tabstop>
  <tabstop>spinBox_drv_para_X_softend_neg</tabstop>
  <tabstop>spinBox_drv_para_X_refpos</tabstop>
  <tabstop>spinBox_drv_para_X_refspeed</tabstop>
  <tabstop>spinBox_drv_para_X_poscor</tabstop>
  <tabstop>spinBox_drv_para_Y_maxspeed</tabstop>
  <tabstop>spinBox_drv_para_Y_softend_pos</tabstop>
  <tabstop>spinBox_drv_para_Y_softend_neg</tabstop>
  <tabstop>spinBox_drv_para_Y_refpos</tabstop>
  <tabstop>spinBox_drv_para_Y_refspeed</tabstop>
  <tabstop>spinBox_drv_para_Y_poscor</tabstop>
  <tabstop>spinBox_drv_para_Z_maxspeed</tabstop>
  <tabstop>spinBox_drv_para_Z_softend_pos</tabstop>
  <tabstop>spinBox_drv_para_Z_softend_neg</tabstop>
  <tabstop>spinBox_drv_para_Z_refpos</tabstop>
  <tabstop>spinBox_drv_para_Z_refspeed</tabstop>
  <tabstop>spinBox_drv_para_Z_poscor</tabstop>
  <tabstop>pushButton_loadparam</tabstop>
  <tabstop>pushButton_saveparam</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>driveparams</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
